scripts/superdesk/keyboard/keyboard.js

```diff
@@ -330,6 +330,37 @@
                 elt['on' + type] = false;
             }
         };
+
+        this.registerShortcut = function (hotkey, callback, options, elem) {
+            this.register(hotkey[0], gettext(hotkey[2]), hotkey[1]);
```

Comment:
Yep, this is populating the help screen. My idea was to call shortcut registration just on one place, and best on the button, because it will trigger click, and we don't need to define double functions for them. If there is need for some additional functions, I created callback for it.